src/Examples.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+import * as Snippet from "./Snippet";
+
+import * as Simple from "./examples/Simple";
+
+interface IExample {
+    lang: string;
+    code: string;
+}
+
+const sources = {
+  "simple": Simple,
+};
+
+function ExampleNames() {
+  return Object.keys(sources);
+}
+
+function ParseExample(ex: IExample): Snippet.ISnippetProps {
+    return {
+        blocks: [{code: "def main():", good: false}] as Snippet.IBlock[],
+        lang: "python",
+    };
+}
+
+function GetExample(name: string) {
+    return ParseExample(sources[name]);
+}
+
+class Example extends React.Component<RouteComponentProps<any>, {}> {
+  constructor(props: RouteComponentProps) {
+    super(props);
+  }
+
+  public render() {
+    const name = this.props.match.params.name;
+    if (name in sources) {
+      return (
+        <Snippet.Snippet {...GetExample("simple")} />
```

Comment:
Shouldn't this be `{...GetExample(name)}`?

Reply:
Technically yeah, that change is in the PR that includes parsing but I can move it here